.github/workflows/pr_welcome.yml

```
@@ -0,0 +1,31 @@
+name: PR Greetings
+
+on: [pull_request]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        pr-message: >+
+          Thank you for opening your first PR into Matplotlib!
+
+          If you have not heard from us in a while please feel free to
+          ping `@matplotlib/developers` or anyone who has commented on the
+          PR.  Most of our reviewers are volunteers and it is easy for
```

```suggestion
          PR.  Most of our reviewers are volunteers and sometimes
```

Sounds a little more professional to me 😃 .